CONTRIBUTING.md

```diff
@@ -30,7 +40,7 @@ asking for suggestions on how to address the documentation part.
 - [Go](https://golang.org/doc/install)
 - [Dep](https://github.com/golang/dep)
 
-### Getting the Repository ###
+### Getting the Repository <a name="Getting_the_Repository"></a> ###
```

Comment:
GitHub already adds anchors to all the titles, so this is unnecessary.

Reply:
Noted.  Thanks